### ISSUE TITLE
Updated BIOCODE table for Gulf of Mexico

### ISSUE THREAD

**Issue by mpinsky** [AquaAuma/FishGlob_data#46]
Melissa Karp (NMFS) reports that the BIOCODES table for the Gulf of Mexico surveys has missing data and errors in it, and that Jeff Rester is working on fixes. She will share it when she learns more as part of the DISMAP update in 2024 spring. We should check whether our version has these issues.

**Comment by jepa** [comment]
@afredston will contact Melissa to move this forward

**Comment by mpinsky** [comment]
@afredston @zoekitchel @jepa , do any of you know why we set `num_cpua = num_cpue` and `wgt_cpua = wgt_cpue` in many and maybe all of the North American scripts (eg, [here](https://github.com/fishglob/FishGlob_data/blob/main/cleaning_codes/get_ai.R#L299) for AI)?

cpua is supposed to be catch per unit area, whereas cpue is catch per unit time. I don't expect those to be equal.

**Comment by zoekitchel** [comment]
> I think I understand now. The code first defines cpue in catch per unit area, then renames it to cpua later. It also defines catch_h as catch per unit time, then renames to cpue later.

Yes! You're right. The cleaning scripts have gotten a bit messy over time. We changed desired column names at one point, so it led to the syntax you observed.

**Comment by mpinsky** [comment]
@zoekitchel , thanks!! Any chance you can review the GMEX pull request? If it looks good, you can approve it. Merging that branch will close this issue.
